core/src/main/java/org/fao/geonet/kernel/KeywordBean.java

```diff
@@ -187,7 +187,7 @@ private static Element createThesaurusNameElt(KeywordBean kb) {
 
             Element ciDateDatetypeEl = new Element("dateType", Namespaces.GMD);
             Element ciDateDatetypeCodeEl = new Element("CI_DateTypeCode", Namespaces.GMD);
-            ciDateDatetypeCodeEl.setAttribute("codeList", "http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/codelist/ML_gmxCodelists.xml#CI_DateTypeCode");
+            ciDateDatetypeCodeEl.setAttribute("codeList", "http://standards.iso.org/iso/19139/resources/gmxCodelists.xml#CI_DateTypeCode");
```

Comment:
A general question for this change, I guess should be fine for non INSPIRE metadata also?

Reply:
As far as it point to the same file (which is not used by GeoNetwork), I guess so.